quadriga/src/main/webapp/WEB-INF/views/errorPage.html

```diff
@@ -0,0 +1,26 @@
+<html layout:decorate="~{layouts/main}">
```

Comment:
the filename should contain 404 in it, since there can be other errors besides 404s that might need an error page.